echopop/utils/load.py

```diff
@@ -386,79 +375,11 @@ def read_validated_data(
             input_dict["acoustics"]["nasc_df"][column_to_add] = df[column_to_add]
     else:
         raise ValueError(
-            """Unexpected data attribute structure. Check API settings located in"""
-            """the configuration YAML and core.py"""
+            "Unexpected data attribute structure. Check API settings located in "
+            "the configuration YAML and core.py."
```

Comment:
I think the configs are more like parameters, so minor suggestion to just leave out the word API.
```suggestion
            "Unexpected data attribute structure. Check the settings in "
            "the configuration YAML and core.py."
```